salt/loader.py

```diff
@@ -815,7 +815,9 @@ def grains(opts, force_refresh=False, proxy=None):
             grains_data.update(ret)
 
     # Run the rest of the grains
-    for key in funcs:
+    # sort keys so that custom grains for proxyenabled modules can overwrite salt-core included grains
+    sorted_keys = sorted(funcs.keys())
```

Comment:
`for key in sorted(funcs.keys()):` sorted_keys is never used again. Also it does not change the readability of the code.

Reply:
Absolutely, was planning to do this. Less lines is better 🐍